src/CompactErrorFormatter.php

```diff
@@ -26,24 +26,24 @@ public function __construct(
 
 	public function formatErrors(
 		AnalysisResult $analysisResult,
-		OutputStyle $style
+		Output $output
```

Comment:
Pretty sure this won't work with PHPStan 0.10 and 0.11. Change the composer constraint to `^0.12`.

Reply:
So improve tests by running any phpstan version to prove it!

Reply:
@enumag is right, that interface was introduced in 0.12, I've updated dependencies accordingly